src/stylesheets/common/_alerts.scss

```diff
@@ -8,6 +8,14 @@
     text-decoration: underline;
     &:hover { color: inherit; }
   }
+  .btn {
+    text-decoration: inherit;
+    white-space: normal;
+  }
+  .btn-mapzen,
```

Comment:
Where is this being used ? ... we wanted to deprecate this and replace it with `.btn-primary` at some point so maybe the right thing to do is also add `.btn-primary` here so that whenever we remove all `.btn-mapzen`s from code we can remove it from the stylesheet easily.